### ISSUE TITLE
bug in parse-mongo-url causing promised-mongo unusable

### ISSUE THREAD

**Issue by tapsboy** [gordonmleigh/promised-mongo#32]
A bug in readPreference value detection in parse-mongo-url, fixed at https://github.com/sorribas/parse-mongo-url/pull/2 but not merged, is rendering promised-mongo unusable.

Any workarounds without forking?

Update: This is not fixed in the underlying repo. I have sent a https://github.com/gordonmleigh/promised-mongo/pull/33 with the updated package.json

**Comment by RangerMauve** [comment]
Is that causing "auth failed" errors with Mongo 3.0? I'm currently ripping my hair out trying to figure out why my production servers are all crashed. :P

**Comment by tapsboy** [comment]
nope.. it doesn't even connect ... citing something like "Invalid readPreference value"
